module/x/peggy/client/cli/tx.go

```diff
@@ -157,3 +159,65 @@ func CmdRequestBatch() *cobra.Command {
 		},
 	}
 }
+
+func CmdSetOrchestratorAddress() *cobra.Command {
+	return &cobra.Command{
+		Use:   "set-orchestrator-address [orchestrator-address] [ethereum-address]",
+		Short: "Allows validators to delegate their voting responsibilities to a given key.",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+			validatorAddr := cliCtx.GetFromAddress()
+
+			msg := types.MsgSetOrchestratorAddress{
+				Validator:    validatorAddr.String(),
```

Comment:
this is an sdk.AccAddress needs to be an sdk.ValAddress. I believe it should be safe just to cast it in this specific case. But @jackzampolin  would know better.

Reply:
Okay, I changed it to just be input directly in the CLI, so it'll require the sender to correctly input the validator address. Maybe Jack knows which would be the canonical approach though